### ISSUE TITLE
code fences line numbers align incorrectly

### ISSUE THREAD

**Issue by TosakRin** [adrian-fuertes/typora-notion-theme#7]
Thanks for your graceful theme and I feel quite motivated every time I use it.

But I encountered a bug: 
when the typora window's width is short enough(specifically, when the code fences line numbers will hide automatically), after clicking the code fences, **the code fences line numbers will always align incorrectly**, as shown in the figure below
![image](https://user-images.githubusercontent.com/52925755/160161673-abcc3a08-acd4-4007-8313-7bda7dbdf8ec.png)

**I only test it on Windows**. Wish you can spare some time to fix it. 

Thanks a lot！

**Comment by TosakRin** [comment]
Forgive me if my description is not clear

After maximize the window, the line numbers will display like this: 

![image](https://user-images.githubusercontent.com/52925755/160162591-6d6fa58c-29f1-4041-b951-e9e1a7fb013d.png)

**Comment by adrian-fuertes** [comment]
Hey @TosakRin! Thank you so much for your issue. I will try to reproduce it and fix it :)

I haven't tried to reproduce it yet, but I don't understand how this could be caused by the theme. Do you not get this issue with other themes?

**Comment by adrian-fuertes** [comment]
Hey @TosakRin! Thank you so much for your issue. I will try to reproduce it and fix it :)

I haven't tried to reproduce it yet, but I don't understand how this could be caused by the theme. Do you now get this issue with other themes?

**Comment by TosakRin** [comment]
> Hey @TosakRin! Thank you so much for your issue. I will try to reproduce it and fix it :)
> 
> I haven't tried to reproduce it yet, but I don't understand how this could be caused by the theme. Do you not get this issue with other themes?

Sorry I didn't reply in time.

I have tried 10+ other themes(including the offical and third-party ones). 

**it seems that the main problem is the aligning bug in maximized window**: Other themes display the same as first figure I posted above if window's width is short. So I think it's not a bug.

**but other themes all work well after maximized window and the line numbers align correctly, which is different from the situation like the second figure I posted above**

I'm sorry that I haven't studied CSS style systematically so I can't make some useful suggestions for yor :(. But some themes seems to choose not to hide line numbers automatically. Maybe it's an alternative solution?

Anyway, thank you so much for your attention to my issus :)

**Comment by adrian-fuertes** [comment]
Hi @TosakRin! I tried to replicate your issue in both Windows 10 and MacOS using `Typora 1.2.4`, but I was unsuccessful. In my testing the code fences always resize adequately. Even if I was able to reproduce this, I don't think the theme can really do anything about it, since it's essentially just styling the text reproduction.

I would recommend updating to the latest Typora version and trying again :)

I'm sorry I couldn't fix your problem and thank you very much for the feedback! I will be closing this issue.
